content/blog/2021/09/camel-quarkus-release-2.2.0/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Camel Quarkus 2.2.0 Released"
+date: 2021-09-01
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 2.2.0 Released"
+summary: "Camel Quarkus 2.2.0 brings Quarkus 2.2, better test coverage and documentation"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 2.2.0 of Camel Quarkus.
+It brings Quarkus 2.2, one extension newly supported in native mode, better test coverage and documentation.
+Many thanks to all contributors and issue reporters!
+
+## Quarkus 2.2
+
+Quarkus 2.2 is focusing mainly on fix issues, improving usability and documentation.
+
+The highlights:
+
+* GraalVM 21.2 now recommended
+* New global flag `quarkus.devservices.enabled=false` to disable Dev Services
+* Enhanced thread model for RESTEasy Reactive
+* MongoDB service binding
+* New extension for running Narayana LRA participants
+
+Please refer to Quarkus 2.2 announcements for more details:
+* [Quarkus 2.2.1.Final](https://quarkus.io/blog/quarkus-2-2-1-final-released/)
+* [Quarkus 2.2.2.Final](https://quarkus.io/blog/quarkus-2-2-2-final-released/)
+
+## Extensions new supported in native mode
+
+* [Language](/camel-quarkus/latest/reference/extensions/language.html)
+
+## Test coverage and closing functionality gaps
+
+In this release, we continued in our efforts from 2.0.0 and 2.1.0 releases to review existing tests and add new ones
+to cover all major use case mentioned in the main Camel documentation. Check this [this GitHub issues query](https://github.com/apache/camel-quarkus/issues?q=is%3Aissue+label%3Aintegration-test+milestone%3A2.2.0+is%3Aclosed) for more details.
+
+## Documentation
+
+We have documented the [new way of managing dependencies]](/camel-quarkus/latest/user-guide/dependency-management.html#_quarkus_tooling_for_starting_a_new_project) in projects generated by [code.quarkus.io](https://code.quarkus.io/) and other [Quarkus tools](https://quarkus.io/guides/tooling).
+
+## Serialization in native mode
+
+GraalVM did not support serialitation for quite long.
+As a result of that some features of Camel components requiring it did not work.
+We made our best to document those gaps at least.
+
+Since GraalVM started supporting in version 21.0, we are continuously working on improving the serialization support.
+In this release we were able to close [two](https://github.com/apache/camel-quarkus/issues/1433) [such](https://github.com/apache/camel-quarkus/issues/1868) long standing issues.
+
+## Breaking changes and migration steps
+
+Please refer to our [2.2.0 Migration guide](/camel-quarkus/latest/migration-guide/2.2.0.html).
+
+## Release notes
+
+Fixed issues:
+
+* [Milestone 2.2.0](https://github.com/apache/camel-quarkus/milestone/15?closed=1)
```

Comment:
2.2.0 must be milestone 18 ? https://github.com/apache/camel-quarkus/milestone/18?closed=1

Reply:
Indeed, a leftover from the base post. Sorry.